bigbluebutton-html5/imports/ui/components/actions-bar/emoji-menu/component.jsx

```diff
@@ -146,72 +147,21 @@ const EmojiMenu = ({
-          tabIndex={-1}
-        />
-        <DropdownListItem
-          icon="happy"
-          label={intl.formatMessage(intlMessages.happyLabel)}
-          description={intl.formatMessage(intlMessages.happyDesc)}
-          onClick={() => actions.setEmojiHandler('happy')}
-          tabIndex={-1}
-        />
-        <DropdownListItem
-          icon="undecided"
-          label={intl.formatMessage(intlMessages.undecidedLabel)}
-          description={intl.formatMessage(intlMessages.undecidedDesc)}
-          onClick={() => actions.setEmojiHandler('neutral')}
-          tabIndex={-1}
-        />
-        <DropdownListItem
-          icon="sad"
-          label={intl.formatMessage(intlMessages.sadLabel)}
-          description={intl.formatMessage(intlMessages.sadDesc)}
-          onClick={() => actions.setEmojiHandler('sad')}
-          tabIndex={-1}
-        />
-        <DropdownListItem
-          icon="confused"
-          label={intl.formatMessage(intlMessages.confusedLabel)}
-          description={intl.formatMessage(intlMessages.confusedDesc)}
-          onClick={() => actions.setEmojiHandler('confused')}
-          tabIndex={-1}
-        />
-        <DropdownListItem
-          icon="time"
-          label={intl.formatMessage(intlMessages.awayLabel)}
-          description={intl.formatMessage(intlMessages.awayDesc)}
-          onClick={() => actions.setEmojiHandler('away')}
-          tabIndex={-1}
-        />
-        <DropdownListItem
-          icon="thumbs_up"
-          label={intl.formatMessage(intlMessages.thumbsUpLabel)}
-          description={intl.formatMessage(intlMessages.thumbsUpDesc)}
-          onClick={() => actions.setEmojiHandler('thumbsUp')}
-          tabIndex={-1}
-        />
-        <DropdownListItem
-          icon="thumbs_down"
-          label={intl.formatMessage(intlMessages.thumbsDownLabel)}
-          description={intl.formatMessage(intlMessages.thumbsDownDesc)}
-          onClick={() => actions.setEmojiHandler('thumbsDown')}
-          tabIndex={-1}
-        />
-        <DropdownListItem
-          icon="applause"
-          label={intl.formatMessage(intlMessages.applauseLabel)}
-          description={intl.formatMessage(intlMessages.applauseDesc)}
-          onClick={() => actions.setEmojiHandler('applause')}
-          tabIndex={-1}
-        />
+        {
+          Object.keys(EMOJI_STATUSES).map(status => (
+            <DropdownListItem
+              className={userEmojiStatus === status ? styles.emojiSelected : null}
+              icon={EMOJI_STATUSES[status]}
+              label={intl.formatMessage(intlMessages[`${status}Label`])}
+              description={intl.formatMessage(intlMessages[`${status}Desc`])}
+              onClick={() => actions.setEmojiHandler(status)}
+              tabIndex={-1}
+            />
+          ))
+        }
         <DropdownListSeparator />
         <DropdownListItem
-          icon="clear_status"
+          icon={CLEAR_STATUS.none}
```

Comment:
Why is it `none`..? It's a "clear_status" icon.
Is there a reason for this string to be moved to the "statuses.js"?